.github/files/lint-project-structure.sh

```diff
@@ -458,6 +458,14 @@ for PROJECT in projects/*/*; do
 		done < <( jq --stream -r 'if length == 2 and .[0][:-1] == ["extra","dependencies","test-only"] then [input_line_number,.[1]] | @tsv else empty end' "$PROJECT/composer.json" )
 	fi
 
+	# - No direct use of WorDBless.
+	if jq -e '.require["automattic/wordbless"] // .["require-dev"]["automattic/wordbless"]' "$PROJECT/composer.json" >/dev/null; then
+		while IFS=$'\t' read -r LINE; do
+			EXIT=1
+			echo "::error file=$PROJECT/composer.json,line=${LINE}::Do not use \`automattic/wordbless\` directly, use \`automattic/jetpack-test-environment\` instead. See #41057 for details."
```

Comment:
Over-the-top nitpick.

```suggestion
			echo "::error file=$PROJECT/composer.json,line=${LINE}::Do not use \`automattic/wordbless\` directly; use \`automattic/jetpack-test-environment\` instead. See #41057 for details."
```